src/main/kotlin/no/nav/familie/ef/mottak/service/OppgaveService.kt

```diff
@@ -61,7 +62,7 @@ class OppgaveService(private val integrasjonerClient: IntegrasjonerClient,
         try {
             log.info("journalPost=$journalpostId finnesBehandlingForPerson=$finnesBehandlingForPerson")
             val behandlesAvApplikasjon =
-                    if (finnesBehandlingForPerson) BehandlesAvApplikasjon.UAVKLART else BehandlesAvApplikasjon.INFOTRYGD
+                    if (finnesBehandlingForPerson) UAVKLART else BehandlesAvApplikasjon.INFOTRYGD
```

Comment:
Kunne kanskje importert BehandlesAvApplikasjon.INFOTRYGD også (når vi gjøre det for UAVKLART?)

Reply:
Det ligger allerede en konstant `INFOTRYGD` (fagsystemkoden) i pakken, så det blir krasj uten `BehandlesAvApplikasjon` :)